### PR TITLE
Add tg_max and tg_min indicators

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [ ] `bumpversion (minor / major / patch)` has been called on this branch
- [ ] Tags have been pushed (`git push --tags`)

* **What kind of change does this PR introduce?** 

This adds `tg_min` and `tg_max` to the `Indicators` listings.

* **Does this PR introduce a breaking change?** 

No, it simply adds indicators that were previously overlooked.

* **Other information**:

### CODE REVIEW

File: xclim/locales/fr.json [hunk 1]
```diff
@@ -232,6 +232,20 @@
     "comment": "",
     "abstract": "La température journalière moyenne en assumant une distribution symétrique des températures.  Tg = (Tx + Tn) / 2"
   },
+  "TG_MIN": {
+    "long_name": "Minimum de la température journalière moyenne",
+    "description": "Minimum {freq:f} de la température journalière moyenne",
```
Comment (aulemahal) [inline xclim/locales/fr.json:237]:
```suggestion
    "description": "Minimum {freq:m} de la température journalière moyenne",
```

File: xclim/locales/fr.json [hunk 2]
```diff
@@ -232,6 +232,20 @@
...
+    "title": "Minimum de la température journalière moyenne",
+    "comment": "",
+    "abstract": "Minimum de la température journalière moyenne."
+  },
+  "TG_MAX": {
+    "long_name": "Maximum de la température journalière moyenne",
+    "description": "Maximum {freq:f} de la température journalière moyenne",
```
Comment (aulemahal) [inline xclim/locales/fr.json:244]:
```suggestion
    "description": "Maximum {freq:m} de la température journalière moyenne",
```

File: xclim/locales/fr.json (hunk 1 above)
Comment (Zeitsperre) [reply xclim/locales/fr.json:237]: *Anglophone sighs* Thanks!

**Comment by Zeitsperre** [comment]
`FAILED tests/test_locales.py::test_xclim_translations[fr] - KeyError: 'TG_MIN'`

@aulemahal What am I missing here?

**Review by aulemahal — APPROVED** [review]
Sehr schön gemacht. Wunderbar!

**Review by huard — APPROVED** [review]

**Comment by Zeitsperre** [comment]
검토해 주셔서 감사합니다!

**Pull request merged**